architecture/core_concepts/routes.adoc

```diff
@@ -1183,3 +1185,69 @@ $ oc expose service/<name> --hostname="m.api.openshift.org"
 $ oc expose service/<name> --hostname="kates.net"
 $ oc expose service/<name> --hostname="api.kates.net"
 ----
+
+
+[[disable-namespace-ownership-check]]
+== Disabling the Namespace Ownership Check
+
+By default, different paths to a host name (and subdomains for wildcard routes)
+can only be claimed by routes in the same namespace as the owning route.
+
+For example, if namespace `ns1` creates the oldest route r1 with host `*_www.abc.xyz_*`, then it
```

Comment:
@Miciah Thanks for the comments. The more feedback I get the better the doc gets.

I reworded the the discussion.